tests/components/dsmr/test_config_flow.py

```diff
@@ -475,7 +475,6 @@ async def test_options_flow(hass: HomeAssistant) -> None:
         "port": "/dev/ttyUSB0",
         "dsmr_version": "2.2",
         "precision": 4,
-        "reconnect_interval": 30,
```

Comment:
This options are removed in the tests as they were only used for testing. The new fixture patches DEFAULT_RECONNECT_INTERVAL instead.

Reply:
Removed these changes in this PR to be addressed in a follow up.